src/mongocxx/test/spec/change_stream.cpp

```diff
@@ -197,6 +197,9 @@ TEST_CASE("Change stream spec tests", "[change_stream_spec]") {
         return;
     }
 
-    run_tests_in_suite("CHANGE_STREAM_TESTS_PATH", &run_change_stream_tests_in_file);
+    auto tests_path = std::getenv("CHANGE_STREAM_TESTS_PATH")
+                          ? std::getenv("CHANGE_STREAM_TESTS_PATH")
+                          : std::string{MONGOCXX_SOURCE_DIR} + "/data/change_stream";
```

Comment:
There are a few other missing `require (!path.empty())`. Instead of fixing them, perhaps it's best to have a utility function in `src/mongocxx/test/spec/util.hh` to return a path given the environment variable name? It doesn't save much code, but it might be nice to have all of the environment variable path handling in one spot. (At some point we could remove the environment variables entirely, or just have one `DATA_PATH`).